### DISCUSSION TITLE
Custom maps

### DISCUSSION THREAD

**Discussion by hjbout** [Budibase/budibase#5312] [Ideas]
Now that we have an awesome embedded map element I would like to request an even more awesome feature. It would be very nice to be able to use your own maps. Think of office layouts (desk reservation app) or factory plans (maintenance app). It is much the same as the current map element. You should be able to upload your own image (or use an attachment?). Map coordinates in pixels?

**Comment by shogunpurple** [comment]
Hi @hjbout - converted this to a discussion as it's more appropriate.

You can do this already in budibase if you use TileLayer, and it's one of the reasons that we went with the open source and extensible LeafletJS rather than google maps/mapbox - https://docs.budibase.com/docs/map#changing-the-map-tileset

However to just use a single image that you can just paste a URL to requires a slight update to the map component, which I think is a good idea. It would allow you to do exactly what you mentioned - just use an image with simple coordinates for your maps.

**Reply by julienpf** [reply]
Hello, I have the topic. With the Embedded Map component I want to use a custom image but the image is replicated (like a wallpaper). Can you help me please ?
